src/components/RepoDetails.jsx

```diff
@@ -307,32 +312,40 @@ function RepoDetails() {
                   </Stack>
                 </Grid>
                 <Grid item xs={4}>
-                  <Typography variant="body1" sx={{ color: '#52637A', fontSize: '1rem' }}>
-                    Copy and pull to pull this image
-                  </Typography>
+                  <Stack direction="row">
```

Comment:
The pull command should be in the TagDetails page as opposed to the RepoDetails page. We also need to forward this comment to UX so they can update the mocks accordingly.